streampark-common/src/main/java/org/apache/streampark/common/enums/ApplicationType.java

```diff
@@ -43,10 +45,13 @@ public int getType() {
     return type;
   }
 
+  @Nonnull
   public String getName() {
     return name;
   }
 
+  /** switch param use this, can't be null */
+  @Nonnull
```

Comment:
```suggestion
  @Nullable
``` 
due to the line `61`